index.html

```diff
@@ -170,6 +170,10 @@
   </div>
 </div>
 
+<p>
+  If you’d like to view last year’s conference sessions and arcade titles, visit <a href="2015.differentgames.org">2015.differentgames.org</a>. For more information about Different Games Collective, please visit <a href="differentgames.org">differentgames.org</a>.
```

Comment:
Also, the tests are saying that you need an `http://` in front of your links here.